os/configuring-dns.md

```diff
@@ -0,0 +1,51 @@
+# DNS configuration
+
+By default, DNS resolution on CoreOS is handled through `/etc/resolv.conf`, which is a symlink to `/run/systemd/resolve/resolv.conf`.
+This file is managed by [systemd-resolved][systemd-resolved].
+Normally, `systemd-resolved` gets DNS IP addresses from [systemd-networkd][systemd-networkd], either via DHCP or static configuration.
+DNS IP addresses can also be set via `systemd-resolved`'s [resolved.conf][resolved.conf].
+See [Network configuration with networkd]({{site.baseurl}}/docs/os/network-config-with-networkd) for more information on `systemd-networkd`.
+
+## Using a local DNS cache
+
+Systemd-resolved includes a DNS stub resolver.
```

Comment:
i echo'd what the systemd-resolved manual says. maybe best to drop 'stub' and simply claim it is a caching dns resolver.